### PR TITLE
Shrutiius

### PR DESCRIPTION
**Pull request by shruvar**

### CODE REVIEW

File: src/pages/Policy/policy.jsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+      <Container maxWidth="lg" style={{ ...containerStyle, ...mainContentStyle }}>
+        <Paper elevation={3} style={{ width: '100%' }}>
+          <Box p={4}>
+            <Typography variant="h4" style={{ ...h1Style, textAlign: 'center', fontFamily: 'Arial Black', fontWeight: 'bold' }}>
+              Privacy Policy
+            </Typography>
+            <br />
+          
+            <Typography variant="body1" style={pStyle}>
+              PulseZest ("we," "us," "our") is committed to protecting your privacy. This Privacy Policy explains how we collect, use, disclose, and safeguard your information when you use our applications, including but not limited to the School Application, Teaching Application, Local Shopkeeper Application, and any other applications we may develop in the future (collectively, the "Applications"). Please read this Privacy Policy carefully. If you do not agree with the terms of this Privacy Policy, please do not access the Applications.
```
Comment (PulseZest) [inline src/pages/Policy/policy.jsx:62]: Replace the PulseZest to the Code Given Below

`<Link to '/'> _PulseZest_<Link/>`

File: src/pages/Policy/terms.jsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            </Typography>
+            <br />
+            <Typography variant="body1" style={pStyle}>
+              Welcome to <Link to ='/'>PulseZest!</Link>  These Terms and Conditions (“Terms”) govern your use of our applications, including but not limited to the School Application, Teaching Application, Local Shopkeeper Application, and any other applications we may develop in the future. By accessing or using any of our applications, you agree to abide by these Terms. Please read them carefully.
+            </Typography>
+            <Typography variant="h5" style={{ ...h2Style, fontFamily: 'Arial Black', fontWeight: 'bold' }}>
+              Acceptance of Terms
+            </Typography>
+            <Typography variant="body1" style={pStyle}>
+              By accessing or using any of the applications provided by PulseZest, you agree to be bound by these Terms and Conditions.
```
Comment (PulseZest) [inline src/pages/Policy/terms.jsx:68]: Replace the PulseZest to the Code Given Below

`<Link to '/'> _PulseZest_<Link/>`

File: src/pages/Policy/terms.jsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+            </Typography>
+            <Typography variant="h5" style={{ ...h2Style, fontFamily: 'Arial Black', fontWeight: 'bold' }}>
+              Application Use
+            </Typography>
+            <Typography variant="body1" style={pStyle}>
+              PulseZest offers various applications, each with its own set of features and functionalities. Users must adhere to the specific terms and conditions of each application they use.
+              PulseZest reserves the right to modify, suspend, or discontinue any application or service at any time without prior notice.
```
Comment (PulseZest) [inline src/pages/Policy/terms.jsx:75]: Replace the PulseZest to the Code Given Below

`<Link to '/'> _PulseZest_<Link/>`

File: src/pages/Policy/terms.jsx [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+              Acceptance of Terms
+            </Typography>
+            <Typography variant="body1" style={pStyle}>
+              By accessing or using any of the applications provided by PulseZest, you agree to be bound by these Terms and Conditions.
+            </Typography>
+            <Typography variant="h5" style={{ ...h2Style, fontFamily: 'Arial Black', fontWeight: 'bold' }}>
+              Application Use
+            </Typography>
+            <Typography variant="body1" style={pStyle}>
+              PulseZest offers various applications, each with its own set of features and functionalities. Users must adhere to the specific terms and conditions of each application they use.
```
Comment (PulseZest) [inline src/pages/Policy/terms.jsx:74]: Replace the PulseZest to the Code Given Below

`<Link to '/'> _PulseZest_<Link/>`

change the color of the PulseZest Everywhere if Possible .

File: src/pages/Policy/terms.jsx [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+            </Typography>
+            <Typography variant="body1" style={pStyle}>
+              While using any PulseZest application, users must not engage in any unlawful, offensive, or harmful behavior. This includes but is not limited to, harassment, defamation, or any activity that violates our policies.
+              Users must respect the rights and privacy of others and must not use our applications to infringe upon the intellectual property rights of others.
+            </Typography>
+            <Typography variant="h5" style={{ ...h2Style, fontFamily: 'Arial Black', fontWeight: 'bold' }}>
+              Privacy
+            </Typography>
+            <Typography variant="body1" style={pStyle}>
+              PulseZest respects user privacy and handles personal information in accordance with our Privacy Policy. Please review the Privacy Policy for each application to understand how your data will be handled.
```
Comment (PulseZest) [inline src/pages/Policy/terms.jsx:95]: Replace the PulseZest to the Code Given Below

`<Link to '/'> _PulseZest_<Link/>`

File: src/pages/Policy/terms.jsx [hunk 6]
```diff
@@ -0,0 +1,149 @@
+import { Box, Container, Paper, Typography } from '@mui/material';
```
Comment (PulseZest) [inline src/pages/Policy/terms.jsx:1]: Create a Separate Folder of Terms. you put the policy.jsx and terms.jsx in the same folder.

**Review by PulseZest — CHANGES_REQUESTED** [review]
1. add the Link to the PulseZest text and change the color of the PulseZest if posisble. 
 

`<Link to '/'> _PulseZest_<Link/>`

2. Create the seprate folder for terms and policy

**Review by Techchauhan — APPROVED** [review]

**Review by PulseZest — APPROVED** [review]

**Pull request merged**